src/codecs/browser-webp/decoder.ts

```diff
@@ -0,0 +1,15 @@
+import { canDecode, fileToBitmap } from '../../lib/util';
+
+export const name = 'Browser WebP Decoder';
+export const supportedExtensions = ['webp'];
+export const supportedMimeTypes = ['image/webp'];
+export async function decode(file: File): Promise<ImageBitmap> {
+  return fileToBitmap(file);
+}
+
+// tslint:disable-next-line:max-line-length It’s a data URL. Whatcha gonna do?
+const webpFile = 'data:image/webp;base64,UklGRkAAAABXRUJQVlA4WAoAAAAQAAAAAAAAAAAAQUxQSAIAAAAAAFZQOCAYAAAAMAEAnQEqAQABAAFAJiWkAANwAP79NmgA';
```

Comment:
This one from Modernizr seems smaller: `'data:image/webp;base64,UklGRh4AAABXRUJQVlA4TBEAAAAvAAAAAAfQ//73v/+BiOh/AAA='`.

https://github.com/Modernizr/Modernizr/blob/400db4043c22af98d46e1d2b9cbc5cb062791192/feature-detects/img/webp.js#L52